docs/datasets/pcd.md

```diff
@@ -0,0 +1,60 @@
+# Precursor Coincident Dataset (PCD)
+
+This page describes the metadata for the Precursor Coincident Dataset (PCD),
+which consists of 13 high-quality 3D measurement sites across the contiguous
+United States (CONUS).
+
+### Download the Data
+
+To download and generate the dataset assets in both GeoJSON (`.geojson`) and
+GeoParquet (`.geoparquet`) formats, run the following command from the root of
+the repository:
+
+```bash
+pixi run python src/coincident/scripts/generate_pcd.py
```

Comment:
You can also add this as a pixi task to simplify (`pixi run generate_pcd`) - https://github.com/uw-cryo/coincident/blob/827e42eea21bf80255518265197594712a223ee6/pyproject.toml#L251-L252